src/index/IndexImpl.Text.cpp

```diff
@@ -803,7 +803,31 @@ Index::WordEntityPostings IndexImpl::readWordCl(
 }
 
 // _____________________________________________________________________________
-Index::WordEntityPostings IndexImpl::readWordEntityCl(
+IdTable IndexImpl::readWordCl(
+    const TextBlockMetaData& tbmd,
+    const ad_utility::AllocatorWithLimit<Id>& allocator) const {
+  IdTable idTable{allocator};
+  idTable.setNumColumns(2);
```

Comment:
```suggestion
  IdTable idTable{2, allocator};
```
(Can also be applied in some other places where you use this pattern)